presto-main/src/main/java/com/facebook/presto/event/QueryMonitor.java

```diff
@@ -43,7 +43,7 @@
 import com.facebook.presto.spi.eventlistener.QueryMetadata;
 import com.facebook.presto.spi.eventlistener.QueryOutputMetadata;
 import com.facebook.presto.spi.eventlistener.QueryStatistics;
-import com.facebook.presto.spi.eventlistener.StageCpuDistribution;
+import com.facebook.presto.spi.eventlistener.TaskDistribution;
```

Comment:
I'm wondering if it would be useful to keep `Stage` in the name. The data is the distribution of task resource usage within a stage, right? `StageResourceDistribution` maybe?

Reply:
Updated to ResourceDistribution to avoid confusion